ldk/javascript/examples/self-test-loop/src/tests/whisper/react-whisper-utils.tsx

```diff
@@ -74,10 +80,15 @@ function withWrapper(WrappedComponent: React.JSXElementConstructor<TestComponent
 
 export class WhisperTestWrapper {
   onResolve: (value: boolean | PromiseLike<boolean>) => void;
+
```

Comment:
Why add the spaces here?

Reply:
I think this is a default behaviour in (I want to say) eslint or something

Reply:
Huh...weird. I wonder why that's the default?

Reply:
Yes, if you open SelfTestLoop project directly in the vscode, it shows much more eslint issues. https://eslint.org/docs/rules/lines-between-class-members